Update eksctl-karpenter.md (#6465)
weaveworks/eksctl: userdocs/src/usage/eksctl-karpenter.md

    With [v0.17.0](https://karpenter.sh/docs/upgrade-guide/#upgrading-to-v0170) Karpenter’s Helm chart package is now stored in Karpenter’s OCI (Open Container Initiative) registry.
    Eksctl therefore is not supporting lower versions of Karpenter for new cluster creation. Previously created clusters shouldn't be affected by this change.
    If you wish to upgrade your current installation of Karpenter please refer to the [upgrade guide](https://karpenter.sh/docs/upgrade-guide/)

>     You have to be logged out of ECR repositories on helm to be able to pull the OCI artifact by running `helm registry logout public.ecr.aws`, failure to do so will result in a 403 error when trying to pull the chart.

Reviewer: ```suggestion
    You have to be logged out of ECR repositories to be able to pull the OCI artifact by running `helm registry logout public.ecr.aws` or `docker logout public.ecr.aws`, failure to do so will result in a 403 error when trying to pull the chart.
```
